### PR TITLE
[FLINK-11565][docs][Translate the Improve the Website page]

### PR DESCRIPTION
**Pull request by klion26**
Translate the "Improve the Website" page
@wuchong

### CODE REVIEW

File: improve-website.zh.md [hunk 1]
```diff
@@ -2,105 +2,103 @@
...
 ```
 cd flink-web
 git checkout asf-site
 ```
 
-## Directory structure and files
+## 目录结构和文件
 
-Flink's website is written in [Markdown](http://daringfireball.net/projects/markdown/). Markdown is a lightweight markup language which can be translated to HTML. We use [Jekyll](http://jekyllrb.com/) to generate static HTML files from Markdown.
+Flink 的官网是用 [Markdown](http://daringfireball.net/projects/markdown/) 所编写。Markdown是一种轻量级标记语言，可以翻译为 HTML。我们使用 [Jekyll](http://jekyllrb.com/) 生成静态 HTML 文件。
```
Comment (XuQianJin-Stars) [inline improve-website.zh.md]: Markdown是一种轻量级 -> Markdown 是一种轻量级

File: improve-website.zh.md [hunk 2]
```diff
@@ -2,105 +2,103 @@
...
 ```
 
-Please feel free to ask any questions you have on the developer mailing list.
+如果有任何疑问，欢迎在开发者邮件列表中咨询。
 
-## Submit your contribution
+## 提交你的贡献
 
-The Flink project accepts website contributions through the [GitHub Mirror](https://github.com/apache/flink-web) as [Pull Requests](https://help.github.com/articles/using-pull-requests). Pull requests are a simple way of offering a patch by providing a pointer to a code branch that contains the changes.
+Flink项目通过 [GitHub Mirror](https://github.com/apache/flink-web) 作为 [Pull Requests](https://help.github.com/articles/using-pull-requests) 接受网站贡献。Pull requests 是一种通过向特定代码分支提交补丁的简单方法。
```
[the remark this answers is not in the document]
Comment (klion26) [reply improve-website.zh.md]: Nice Catch

File: improve-website.zh.md [hunk 3]
```diff
@@ -2,105 +2,103 @@
...
-2. Push the commit to a dedicated branch of your fork of the Flink repository at GitHub.
+1. 将你的更改提交到本地 git 仓库。 **请确保你的提交不包含自动生成的文件（`content/`目录中的任何文件）。** 如果不是重大重构，请将代码压缩到一个提交中。
+2. 请将提交推送到 GitHub 上你自己仓库的一个特定分支。
 
 	```
 	git push origin myBranch
 	```
-
-3. Go the website of your repository fork (`https://github.com/<your-user-name>/flink-web`) and use the "Create Pull Request" button to start creating a pull request. Make sure that the base fork is `apache/flink-web asf-site` and the head fork selects the branch with your changes. Give the pull request a meaningful description and submit it.
+3. 打开你镜像的仓库(`https://github.com/<your-user-name>/flink-web`) 并使用 "Create Pull Request" 按钮开始创 pull request 取请求。 确保基础分支是`apache/flink-web asf-site` ，并且 head fork 选择带有更改内容的分支。 为 pull request 提供有意义的描述并提交。
```
[the remark this answers is not in the document]
Comment (klion26) [reply improve-website.zh.md]: Nice catch

File: improve-website.zh.md (hunk 1 above)
Comment (klion26) [reply improve-website.zh.md]: Nice Catch

File: improve-website.zh.md [hunk 4]
```diff
@@ -2,105 +2,103 @@
...
+**本节仅适用于提交者。**
 
-### ASF website git repositories
+### ASF 网站的 git 仓库
 {:.no_toc}
 
 **ASF writable**: https://gitbox.apache.org/repos/asf/flink-web.git
 
-Details on how to set the credentials for the ASF git repository are [linked here](https://gitbox.apache.org/).
+有关如何设置 ASF git 仓库凭据的详细信息可以参考 [链接]（https://gitbox.apache.org/）。
```
Comment (XuQianJin-Stars) [inline improve-website.zh.md]: （https://gitbox.apache.org/） ->  (https://gitbox.apache.org/)  ?

File: improve-website.zh.md (hunk 2 above)
Comment (XuQianJin-Stars) [inline improve-website.zh.md]: Flink 项目通过 [GitHub Mirror](https://github.com/apache/flink-web)以[Pull Requests](https://help.github.com/articles/using-pull-requests)方式提交网站贡献。

**Comment by klion26** [comment]
@XuQianJin-Stars thank you for your review, I've updated the doc, PTAL when you are free.

File: improve-website.zh.md [hunk 5]
```diff
@@ -2,105 +2,103 @@
...
 ```
 
-Please feel free to ask any questions you have on the developer mailing list.
+如果有任何疑问，欢迎在开发者邮件列表中咨询。
 
-## Submit your contribution
+## 提交你的贡献
 
-The Flink project accepts website contributions through the [GitHub Mirror](https://github.com/apache/flink-web) as [Pull Requests](https://help.github.com/articles/using-pull-requests). Pull requests are a simple way of offering a patch by providing a pointer to a code branch that contains the changes.
+Flink 项目通过 [GitHub Mirror](https://github.com/apache/flink-web) 作为 [Pull Requests](https://help.github.com/articles/using-pull-requests) 接受网站贡献。Pull requests 是一种通过向特定代码分支提交补丁的简单方法。
```
Comment (XuQianJin-Stars) [inline improve-website.zh.md]: Flink 项目通过 [GitHub Mirror](https://github.com/apache/flink-web)以提交[Pull Requests](https://help.github.com/articles/using-pull-requests)方式接受网站贡献。 ???

**Comment by XuQianJin-Stars** [comment]
LGTM +1

**Comment by wuchong** [comment]
Thanks for the contribution @klion26  and the reviewing @XuQianJin-Stars . 

It looks good to me.

Will merge this,

**Pull request closed without merging**